python.md

```diff
@@ -974,6 +974,49 @@ from path.to.some.module import \
     thing1, thing2, thing3, thing4
 ```
 
+### Make function signatures explicit
```

Comment:
Don't forget to add to the ToC.

Reply:
Thanks, done in https://github.com/octoenergy/conventions/pull/66/commits/f36dc9597993187f9c7493b6e99ece4bf20340cc